docs/prereqs.md

```diff
@@ -0,0 +1,83 @@
+# Prerequisites
+
+The Canary Checker leans heavily on Prometheous integration.  This guide explains how to install the dependencies. 
+
+## A Kubernetes Cluster
+
+This is pretty obvious, but first [install](https://github.com/flanksource/karina) a kubernetes cluster and ensure you are able to run basic `kubectl` commands to create pods, services, and that cluster DNS is functioning properly. 
+
+## Metrics Server
+
+For many environments, Prometheus requires the [Kubernetes Metrics Server](https://github.com/kubernetes-sigs/metrics-server) to get metrics from the worker nodes. How this is deployed is slightly different for different Kubernetes environments. [Here is an example of how to install the Metrics Server on AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/metrics-server.html).  If you are using GKE, you can skip this step as Prometheus is already installed using Stack Driver metrics. 
+
+To ensure you are getting metrics run: 
+
+```
+kubectl get --raw /metrics
+```
+
+
+## Prometheus Operator
+
+The Prometheus Operator is required for Canary Checker. 
```

Comment:
So this isn't strictly true - There are service monitors that are bundled in the deployment file, but other than that there is no dependency.  I also don't think we need to include the installation instructions

Reply:
When I installed I got an error stating service monitor crds were not found that were resolved by installing Prometheus. I will say that it is optional.  I know I got value out of installing the operator installation just because I spent about an hour trying to find the easiest way to install it.  Everywhere I saw suggested helm and I didn't want to use helm.